### PR TITLE
command: add -v example

### PR DESCRIPTION
**Pull request by sedrubal**
- ~~The page (if new), does not already exist in the repo.~~
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- ~~The page description includes a link to documentation or a homepage (if applicable).~~

### CODE REVIEW

File: pages/common/command.md [hunk 1]
```diff
@@ -5,3 +5,7 @@
 - Execute the ls program literally, even if an ls alias exists:
 
 `command {{ls}}`
+
+- Find the path to the executable of the command ls:
```
Comment (navarroaxel) [inline pages/common/command.md:9]: Based on this output:
```
$ command -v ls
alias ls='ls --color=auto'
$ command -v bat
/usr/bin/bat
```
Maybe this is better:
```suggestion
- Display the path to the executable or the alias definition of a specific command:
```

File: pages/common/command.md [hunk 2]
```diff
@@ -5,3 +5,7 @@
...
+
+`command -v {{ls}}`
```
Comment (navarroaxel) [inline pages/common/command.md:11]: `command -v {{command}}` doesn't look good. Maybe:
```suggestion
`command -v {{command_name}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @sedrubal, nice example. I left comments.

**Comment by sedrubal** [comment]
Thanks for the fast review. I changed the mentioned things.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @sedrubal!

**Pull request merged**